Package/gulpfile.js

```diff
@@ -40,6 +40,16 @@ const buildAndroid = async () => {
   exec('./gradlew babylonjs_react-native:assembleRelease', '../Apps/Playground/android');
 };
 
+const makeUWPProject = async () => {
+  exec('.\\..\\Modules\\@babylonjs\\react-native\\windows\\scripts\\Setup.bat');
+}
+
+const buildUWPProject = async () => {
+  exec('.\\..\\Modules\\@babylonjs\\react-native\\windows\\scripts\\Build.bat');
+}
+
+const buildUWP = gulp.series(makeUWPProject, buildUWPProject);
```

Comment:
Today we run this gulp build on Mac agents so we can build both iOS and Android, which means this added stuff for UWP will break. I think we are going to need to rework this so that we build Android, iOS, and UWP on different build agents, publish the built stuff as artifacts, and then have a separate stage that pulls the artifacts and packages them all together into the NPM package. Maybe we just save the changes to gulpfile.js for a future PR?

Reply:
that sounds good. I'll probably go ahead and add the buildUWP gulp task to the exports. But i agree it shouldn't be in the main build task.